core/src/components/searchbar/searchbar.tsx

```diff
@@ -104,7 +104,7 @@ export class Searchbar implements ComponentInterface {
    * Setting to `"always"` shows the cancel button regardless
    * of focus state.
    */
-  @Prop() showCancelButton: boolean | string = 'never';
+  @Prop() showCancelButton = 'never';
```

Comment:
Let's add a proper type here, will help to prevent errors and improve documentation and autocompletion
```suggestion
  @Prop() showCancelButton: 'never' | 'focus' | 'always' = 'never';
```

Reply:
Good idea!